guides/INSTALLING.md

```diff
@@ -246,18 +256,24 @@ Run all of the services in the [web journey](#web-journey), **but** change the c
 and also run the following:
 
 * [florence](https://github.com/ONSdigital/florence) - use: `$ make debug ENCRYPTION_DISABLED=true`
-* [The-Train](https://github.com/ONSdigital/The-Train) - use: `$ ./run.sh`
+* [The-Train](https://github.com/ONSdigital/The-Train)
 * [dp-api-router](https://github.com/ONSdigital/dp-api-router)
 
-Florence will be available at http://localhost:8081/florence/login
+If you also want to run Florence with the ability to edit images on the homepage (for the Featured Content section), you will need to additionally run:
 
-The website will be available at [http://localhost:8081](http://localhost:8081) after a successful login into florence. Login details are in the [florence repository](https://github.com/ONSdigital/florence/blob/develop/USAGE.md).
+* [dp-image-api](https://github.com/ONSdigital/dp-image-api)
+* [dp-image-importer](https://github.com/ONSdigital/dp-image-importer)
```

Comment:
These need to be run with `make debug  ENCRYPTION_DISABLED=true` if other services are also run without encryption. It's also necessary to run the `dp-upload-service` and `dp-download-service` too. Also homepage editing needs florence to have the `ENABLE_HOMEPAGE_PUBLISHING=true` feature flag enabled.